bookkeeper-dist/src/assemble/bin-all.xml

```diff
@@ -0,0 +1,100 @@
+<!--
+   Licensed to the Apache Software Foundation (ASF) under one or more
+   contributor license agreements.  See the NOTICE file distributed with
+   this work for additional information regarding copyright ownership.
+   The ASF licenses this file to You under the Apache License, Version 2.0
+   (the "License"); you may not use this file except in compliance with
+   the License.  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+-->
+<assembly xmlns="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2 http://maven.apache.org/xsd/assembly-1.1.2.xsd">
+  <id>bin</id>
+  <formats>
+    <format>tar.gz</format>
+  </formats>
+  <includeBaseDirectory>true</includeBaseDirectory>
+  <fileSets>
+    <fileSet>
+      <directory>target</directory>
+      <outputDirectory>/</outputDirectory>
+      <includes>
+        <include>${project.artifactId}-${project.version}.jar</include>
+      </includes>
+    </fileSet>
+    <fileSet>
+      <directory>../../bookkeeper-server/conf</directory>
+      <outputDirectory>/conf</outputDirectory>
+    </fileSet>
+    <fileSet>
+      <directory>../../bookkeeper-server/bin</directory>
```

Comment:
It is strange that, in my verify, the bin/bookkeeper for dist is not the same as bookkeeper-server/bin/bookkeeper
```
➜  bookkeeper git:(sijie/add_slim_module) ll bookkeeper-dist/target/bookkeeper-dist-4.6.0-SNAPSHOT/bin 
total 40
-rwxr-xr-x  1 jia  staff   7.0K Oct 17 15:33 bookkeeper
-rwxr-xr-x  1 jia  staff   2.8K May 27 21:21 bookkeeper-cluster.sh
-rwxr-xr-x  1 jia  staff   4.3K May 27 21:21 bookkeeper-daemon.sh
➜  bookkeeper git:(sijie/add_slim_module) ll bookkeeper-server/bin 
total 40
-rwxr-xr-x  1 jia  staff   7.2K Nov 20 19:30 bookkeeper
-rw-r--r--  1 jia  staff   2.8K May 27 21:21 bookkeeper-cluster.sh
-rwxr-xr-x  1 jia  staff   4.3K May 27 21:21 bookkeeper-daemon.sh
```

Reply:
@jiazhai I verified the package. I see it differently. I am wondering if you do a clean install. can you rerun the verification again?

```
[MacBook-Pro bookkeeper-all-4.6.0-SNAPSHOT]$ ls -l bin/
total 56
-rwxr-xr-x  1 sijie  staff  4319 Nov  7 02:47 benchmark
-rwxr-xr-x  1 sijie  staff  7365 Nov 21 11:18 bookkeeper
-rwxr-xr-x  1 sijie  staff  2869 Nov  7 02:47 bookkeeper-cluster.sh
-rwxr-xr-x  1 sijie  staff  4357 Nov  7 02:47 bookkeeper-daemon.sh
[MacBook-Pro bookkeeper-all-4.6.0-SNAPSHOT]$ ls -l ~/Workspaces/bookkeeper/bookkeeper-server/bin/
total 40
-rwxr-xr-x  1 sijie  staff  7365 Nov 21 11:18 bookkeeper
-rw-r--r--  1 sijie  staff  2869 Nov  7 02:47 bookkeeper-cluster.sh
-rwxr-xr-x  1 sijie  staff  4357 Nov  7 02:47 bookkeeper-daemon.sh
[MacBook-Pro bookkeeper-all-4.6.0-SNAPSHOT]$
```

Reply:
Re- verified, not reproduced again. sorry for it. 
This package runs well, will merge it.